recipes/pyscenic/build.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e  
+patch -p1 < ${RECIPE_DIR}/dill_patch.patch
+
+$PYTHON -m pip install . --no-deps --ignore-installed -vv
```

Shellcheck

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)